app/assets/javascripts/application.js

```diff
@@ -1,3 +1,16 @@
+// DO NOT REQUIRE jQuery or jQuery-ujs in this file!
+// DO NOT REQUIRE TREE!
+
+// CRITICAL that generated/vendor-bundle must be BEFORE bootstrap-sprockets and turbolinks
+// since it is exposing jQuery and jQuery-ujs
+//= require react_on_rails
+
+//= require generated/vendor-bundle
+//= require generated/app-bundle
```

Comment:
See [`/client/webpack.client.base.config.js`](https://github.com/shakacode/react_on_rails-generator-results/pull/123/files#diff-7b11f5a99db3faafe8e1a74d1f1d5df0R25) for how we set up the app-bundle.